analyzer/tests/functional/analyze_and_parse/test_files/cppcheck_args.txt

```diff
@@ -0,0 +1 @@
+-UHAVE_NULL_DEREFERENCE
```

Comment:
Can we add comments to this file? A comment would be nice about the fact that this is how you can disable an error in cppcheck.